cmd/server/api/rpcmiddleware_test.go

```diff
@@ -0,0 +1,187 @@
+			handlerToBeCalled:  "standard",
+			storedContracts:    []scan.ERC721UniversalContract{},
+		},
+		{
+			name: "Good request with eth_call method",
+			body: `{
+		    "jsonrpc": "2.0",
+		    "method": "eth_call",
+		    "params": [{
+		        "to": "0x26CB70039FE1bd36b4659858d4c4D0cBcafd743A",
+		        "data": "0x70a082310000000000000000000000001b0b4a597c764400ea157ab84358c8788a89cd28"
+		    }, "latest"],
+		    "id": 1
+		}`,
+			contentType:        "application/json",
+			method:             "POST",
+			expectedStatusCode: http.StatusOK,
+			expectedResponse:   "erc721Handler called",
+			handlerToBeCalled:  "erc721",
+			storedContracts: []scan.ERC721UniversalContract{
+				{
+					Address: common.HexToAddress("0x26CB70039FE1bd36b4659858d4c4D0cBcafd743A"),
+					Block:   uint64(0),
+					BaseURI: "evochain1/collectionId/",
+				},
+			},
+		},
+		{
+			name:               "Good request with eth_call method but no remote minting method",
+			body:               `{"jsonrpc":"2.0","method":"eth_call","params":[{"data":"0x95d89b41","to":"0x26CB70039FE1bd36b4659858d4c4D0cBcafd743A"}],"id":1}`,
+			contentType:        "application/json",
+			method:             "POST",
+			expectedStatusCode: http.StatusOK,
+			expectedResponse:   "standardHandler called",
+			handlerToBeCalled:  "standard",
+			storedContracts: []scan.ERC721UniversalContract{
+				{
+					Address: common.HexToAddress("0x26CB70039FE1bd36b4659858d4c4D0cBcafd743A"),
+					Block:   uint64(0),
+					BaseURI: "evochain1/collectionId/",
+				},
+			},
+		},
+		{
+			name:               "Good request with no erc721 method",
+			body:               `{"method":"eth_getBlockByNumber","params":["latest",false],"id":1,"jsonrpc":"2.0"}`,
+			contentType:        "application/json",
+			method:             "POST",
+			expectedStatusCode: http.StatusOK,
+			expectedResponse:   "standardHandler called",
+			handlerToBeCalled:  "standard",
+		},
+		{
+			name:               "Bad request with GET method",
+			body:               `{"jsonrpc":"2.0","method":"eth_call","params":[{"data":"0x70a082310000000000000000000000001b0b4a597c764400ea157ab84358c8788a89cd28","to":"0x26CB70039FE1bd36b4659858d4c4D0cBcafd743A"}],"id":1}`,
+			contentType:        "application/json",
+			method:             "GET",
+			expectedStatusCode: http.StatusBadRequest,
+			expectedResponse:   "No JSON RPC call or invalid Content-Type\n",
+			handlerToBeCalled:  "none",
+		},
+		{
+			name:               "Bad request with jsonrpc 1.0",
+			body:               `{"jsonrpc":"1.0","method":"eth_call","params":[{"data":"0x70a082310000000000000000000000001b0b4a597c764400ea157ab84358c8788a89cd28","to":"0x26CB70039FE1bd36b4659858d4c4D0cBcafd743A"}],"id":1}`,
+			contentType:        "application/json",
+			method:             "POST",
+			expectedStatusCode: http.StatusBadRequest,
+			expectedResponse:   "Invalid JSON-RPC version\n",
+			handlerToBeCalled:  "none",
+		},
+	}
+
+	// Run tests
+	for _, ttest := range tests {
```

Comment:
Kindly rename it using `tt` and `tt := tt` so that we maintain consistency throughout the entire project.